### ISSUE TITLE
Test data does not match the input check requirements 

### ISSUE THREAD

**Issue by cahuparo** [nf-core/pathogensurveillance#45]
### Description of the bug

Hi @zachary-foster , 

I know you mentioned you were working on fixing the sample check script and converting it to an R script. 

I was testing the pipeline to generate the record_message output for the report. I encounter the following issue:

```nextflow
nextflow.config.Manifest@3a7f7201
[3c/caa1f5] NOTE: Process `NFCORE_PATHOGENSURVEILLANCE:PATHOGENSURVEILLANCE:INPUT_CHECK:SAMPLESHEET_CHECK (metadata_full.csv)` terminated with an error exit status (1) -- Execution is retried (1)
ERROR ~ Error executing process > 'NFCORE_PATHOGENSURVEILLANCE:PATHOGENSURVEILLANCE:INPUT_CHECK:SAMPLESHEET_CHECK (metadata_full.csv)'

Caused by:
  Process `NFCORE_PATHOGENSURVEILLANCE:PATHOGENSURVEILLANCE:INPUT_CHECK:SAMPLESHEET_CHECK (metadata_full.csv)` terminated with an error exit status (1)

Command executed:

  check_samplesheet.py \
      metadata_full.csv \
      samplesheet.valid.csv
  
  cat <<-END_VERSIONS > versions.yml
  "NFCORE_PATHOGENSURVEILLANCE:PATHOGENSURVEILLANCE:INPUT_CHECK:SAMPLESHEET_CHECK":
      python: $(python --version | sed 's/Python //g')
  END_VERSIONS

Command exit status:
  1

Command output:
  (empty)

Command error:
  [CRITICAL] The sample sheet **must** contain these column headers: fastq_1, sample, fastq_2.

Work dir:
  /nfs5/BPP/Grunwald_Lab/home/paradarc/pathogensurveillance/work/ca/00ab1aa2b9a3a2f8e72580320b41e7

Tip: view the complete command output by changing to the process work dir and entering the command `cat .command.out`

 -- Check '.nextflow.log' file for details
``` 

### Command used and terminal output

```console
nextflow run nf-core/pathogensurveillance --input https://raw.githubusercontent.com/grunwaldlab/pathogensurveillance/master/test/data/metadata_small.csv --outdir test_out --download_bakta_db true -profile mamba -resume
```

### System information

CQLS

**Comment by cahuparo** [comment]
We solved this issue. I was using the wrong version of the pipeline. Instead of using the `main.nf` from the local installation:

```
nextflow run main.nf --input https://raw.githubusercontent.com/grunwaldlab/pathogensurveillance/master/test/data/metadata_small.csv --outdir test_out --download_bakta_db true -profile mamba -resume
```

**Comment by masudermann** [comment]
This command worked for me too, with the docker profile. 

`nextflow run main.nf --input https://raw.githubusercontent.com/grunwaldlab/pathogensurveillance/master/test/data/metadata_small.csv --outdir test_out --download_bakta_db true -profile docker -resume`

On my Linux computer, I found I needed to use Docker Engine, not Docker Desktop. I ran into lots of permissions and resource issues otherwise. 

The following instruction was key, and something I missed the first time. 

(https://docs.docker.com/engine/install/linux-postinstall/)
